picking_good_slos.md

```diff
@@ -0,0 +1,57 @@
+# How to pick a good SLO
+
+A Service Level Objective (SLO) is a measurable, specific goal for a given service's reliability level of service over time, based on one or more SLIs.
```

Comment:
To be consistent, I would suggest the expansion of the SLI acronym (like you did for SLOs at the beginning of this sentence).

Also, I don't think it would hurt to actually say what an SLI is even though you are referencing the "Picking good SLIs" document later on.

How about adding something like this to the end of this sentence:

The SLI is a measure of "what-you-want" (e.g. x should be true) and the SLO is the measure of "how-much-you-want-it" (e.g. y% of the time). Ideally, the SLOs/SLIs are reflecting of some aspect of the service which a customer would consider necessary to the satisfactory usage of that service. For example, as a customer, it is really important for me to be able to get an answer back for my search within 5 seconds (what I want) at least 99% of the time (how much I want it). These are called "customer-centric" SLOs and the reflect a customer's view of the "reliability" they expect for that service.